docs/content/manual/manual.yml

```diff
@@ -269,6 +269,13 @@ sections:
         available in the program and has a string whose contents are to the texts
         in the file named `bar`.
 
+      * `--depth n`:
+
+        This option sets the maximum parsing depth (of objects and lists) to `n`.
```

Comment:
maybe array instead of lists?

Reply:
Fixed in [c4e6358](https://github.com/jqlang/jq/pull/3063/commits/c4e63581f00ca5d1886db5723b2cf5607a5c6214).